MIDAS/src/systems.cpp

```diff
@@ -20,8 +20,9 @@ DECLARE_THREAD(data_logger, RocketSystems* arg) {
 
 DECLARE_THREAD(barometer, RocketSystems* arg) {
     while (true) {
-        THREAD_SLEEP(16);
-        //Serial.println("BARO");
+        arg->rocket_state.barometer.update(arg->sensors.barometer.read());
+
+        THREAD_SLEEP(1);
```

Comment:
is this too short is anything else going to actually happen with this short of a delay @redindelible @NPIPHI

Reply:
FreeRTOS's scheduler is advanced enough that all threads should run fine. We will need to tune this once we've got the boards though.

Reply:
got it, will merge